app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,177 @@
+
+
+    <!-- Floating Action Buttons - Titles -->
+
+    <string name="go_to_top">Retour en haut</string>
+    <string name="search_by_tags_or_persons">Recherche par tags ou par personnes</string>
+    <string name="compose">Nouveau message</string>
+
+
+    <string name="search_alert_title">Recherche</string>
+    <string name="search_alert_tag">par tags</string>
+    <string name="search_alert_people">par personne</string>
+    <string name="search_alert_bypeople_validate_needsomedata">Veuillez ajouter un nom</string>
+
+    <!-- Dialog License -->
+
+    <string name="markdown_text">&lt;i>"Sur diaspora* il est possible de mettre en forme votre texte des messages de statut, des commentaires et des conversations en utilisant un système de balises simple nommé Markdown. Cette page est une introduction aux codes utilisés pour créer ce formatage..&lt;/i>&lt;br> &lt;br>
+
+        &lt;b>En-têtes&lt;/b>&lt;br>
+        # Ceci est un très grand titre&lt;br>
+        ## Moitié moins grand que le précédent&lt;br>
+        ### Deux fois la taille d\'un texte normal&lt;br> &lt;br>
+
+        &lt;b>Italique et gras&lt;/b>&lt;br>
+        Italiques : *mot* ou _mot_&lt;br>
+        Gras : **mot** ou __mot__&lt;br>
+        Gras italique : ***mot*** ou ___mot___&lt;br> &lt;br>
+
+        &lt;b>Listes&lt;/b>&lt;br>
+        *, + ou - au début chaque ligne que vous souhaitez lister ou 1., 2., etc. au début de chaque ligne de la liste&lt;br> &lt;br>
+
+        &lt;b>Citation&lt;/b>&lt;br>
+        Lorsque vous souhaitez citer un extrait d\'un article ou d\'un autre commentaire, vous pouvez formater facilement votre texte en commençant la ligne ou le paragraphe avec le caractère >&lt;br> &lt;br>
+
+        &lt;b>Ligne horizontale&lt;/b>&lt;br>
+        Pour créer une ligne horizontale, utilisez au moins trois ---, ___ ou *** sur une ligne à part. N\'importe quel nombre au-delà de trois aura le même résultat et les espaces entre les caractères n\'ont pas d\'importance.&lt;br> &lt;br>
+
+        &lt;b>Liens intégrés&lt;/b>&lt;br>
+        [texte qui apparaîtra ici](http://lien.adresse.ici \"texte alternatif\") Le \"texte alternatif\" est facultatif et affiché lorsque la souris survole le lien.&lt;br> &lt;br>
+
+        &lt;b>Images&lt;/b>&lt;br>
+        ![Texte alternatif](http://siteweb.com/image.jpg \"titre facultatif\") Le texte alternatif entre crochets est affiché si l\'image ne peut pas être chargée et le titre optionnel est affiché lorsque la souris survole l\'image ; les deux sont utiles mais pas essentiels.&lt;br> &lt;br>
+
+        &lt;b>Escape&lt;/b>&lt;br>
+        Si vous voulez inclure dans votre message un caractère aussi utilisé dans le codage Markdown, vous pouvez empêcher qu\'il soit interprété en l\'\"échappant\". Pour cela, placez un backslash (\) devant le caractère. Vous pouvez toutefois écrire "diaspora*" ou "D*" sans que l\'astérisque devienne un code de formatage !&lt;br> &lt;br>
```

Comment:
Mixed escapes and non-escapes. Does this serve any purpose? (I'm relatively new to the project and did not look further into the HTML (?) in the resources.)